src/libraries/System.Data.Odbc/tests/CommandBuilderTests.cs

```diff
@@ -8,7 +8,7 @@ namespace System.Data.Odbc.Tests
 {
     public class CommandBuilderTests : IntegrationTestBase
     {
-        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsThreadingSupported))]
+        [ConditionalFact]
         public void QuoteIdentifier_UseConnection()
```

Comment:
That one looks weird, I think it should also just be a `[Fact]`

Reply:
Oh thanks for the catches... These changes were part of `git revert ...` back to https://github.com/dotnet/runtime/pull/37822/commits. I'm guessing the git revert didn't work as intended?

Reply:
@mdh1418 no, the git revert worked fine. The plain `[ConditionalFact]` were actually introduced a few months ago in https://github.com/dotnet/runtime/commit/3d3ad67f77d01d50e8157b78d9c927f73c5ce032#diff-8d42e0fa04a6c0fee7dbb0950d9b6264R12 😄

Reply:
and it looks like that was actually intentional since the tests fail otherwise with:
```
    System.Data.Odbc.Tests.CommandBuilderTests.QuoteIdentifier_UseConnection [FAIL]
      Microsoft.DotNet.XUnitExtensions.SkipTestException : Dependency unixODBC with minimum version 2.3.1 is required.
      libodbc.2.dylib assembly:<unknown assembly> type:<unknown type> member:(null)
      Stack Trace:
        /_/src/libraries/System.Data.Odbc/tests/IntegrationTestBase.cs(28,0): at System.Data.Odbc.Tests.IntegrationTestBase..ctor()
           at System.Data.Odbc.Tests.CommandBuilderTests..ctor()
        /_/src/mono/netcore/System.Private.CoreLib/src/System/Reflection/RuntimeMethodInfo.cs(860,0): at System.Reflection.RuntimeConstructorInfo.InternalInvoke(Object obj, Object[] parameters, Boolean wrapExceptions)
```

The tests are explicitly throwing `SkipTestException` which is handled by the `ConditionalFact` to skip the test.

Sorry for the additional work but you need to un-revert these back 🤦

Reply:
Yeah they throw SkipTestException in a common path 😄

Reply:
Removed the commit that changed `[ConditionalFact]` to `[Fact]`.